docs/documentation/03-commands.md

```diff
@@ -2,7 +2,17 @@
 
 ![Booster architecture](../img/booster-arch.png)
 
-Commands and Command Handlers define the **write** API of your application (highlighted in yellow in the diagram). Commands are objects that are sent to the `/commands` endpoint.
+Commands and Command Handlers define the **write** API of your application (highlighted in yellow in the diagram). Commands are objects that are sent to the `/commands` endpoint. The request should look more or less like this:
+
+```json
+{
+  "typeName": "CreateCart"  // name of the command class
+  "value": {
+  "cartId": "aaaa",         // contents of the fields of the class
+    ...
+  }
+}
+```
```

Comment:
This is already documented in [here](docs/documentation/09-rest-api.md) although we should improve that documentation.
I think it would be enough to just have a link here pointing to the commands section of the API documentation

Reply:
fixed in https://github.com/boostercloud/booster/pull/6/commits/9bf7c6fbdaf0f25f86fa9d2b74a75ab2058573f7 thanks